Makefile

```diff
@@ -18,8 +18,17 @@ train:
 build:
 	docker build -f Dockerfile -t ${IMAGE_TAG} .
 
+train-in-container: build
+	-docker rm ${CONTAINER_NAME}
+	docker run -it \
+		--name ${CONTAINER_NAME} \
+		-v $(CURDIR)/outputs:/root/outputs \
+		-e "PYTHONPATH=." \
+		${IMAGE_TAG} \
+		python bin/train.py
+
 run: build
-	docker rm ${CONTAINER_NAME} || true
+	-docker rm ${CONTAINER_NAME}
```

Comment:
My assumption is that the `|| true` is to avoid terminating if docker rm fails, pretending a dash should accomplish the same thing.

Reply:
Excellent!